dotnet/src/SemanticKernel.UnitTests/Functions/SemanticFunctionTests.cs

```diff
@@ -317,7 +317,7 @@ public async Task RunAsyncChangeVariableInvokingHandlerAsync()
         };
 
         // Act
-        await sut.RunAsync(function, originalInput);
+        await sut.InvokeAsync(function, originalInput);
```

Comment:
nit

In FunctionTest, the service under test is actually the function and not the kernel.

```suggestion
        await kernel.InvokeAsync(sut, originalInput);
```
OR
```suggestion
        await kernel.InvokeAsync(function, originalInput);
```